src/Services/ActionManager.vala

```diff
@@ -116,17 +118,156 @@ public class Akira.Services.ActionManager : Object {
     }
 
     private void action_open () {
-        warning ("open");
+        var open_dialog = new Gtk.FileChooserDialog ("Pick a file",
```

Comment:
We need to use a NativeFileChooser here

Reply:
Ok, let me rework it!

Reply:
I change into NativeDialog on top of this, but if breaks for me after select a file to open/save. Maybe is due to flatpak. If you can fix it great, by now I will ignore this suggestion, as cannot find any example to implement it.